codeforamerica/vita-min
# Don't include DeductionsFromFAGI if its zero in NC-D400 XML

## Link to pivotal/JIRA issue
- e.g. [link](https://codeforamerica.atlassian.net/jira/software/c/projects/FYST/boards/9?selectedIssue=FYST-1406)
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Donit include DeductionsFromFAGI if its 0
## How to test?
- create an intake where NC D400 line 9 is 0, check that `DeductionsFromFAGI` is not included in the xml
